lib/dictBuilder/cover.c

```diff
@@ -620,14 +641,24 @@ static size_t COVER_ctx_init(COVER_ctx_t *ctx, const void *samplesBuffer,
     for (i = 0; i < ctx->suffixSize; ++i) {
       ctx->suffix[i] = i;
     }
-    /* qsort doesn't take an opaque pointer, so pass as a global.
-     * On OpenBSD qsort() is not guaranteed to be stable, their mergesort() is.
-     */
-    g_coverCtx = ctx;
-#if defined(__OpenBSD__)
+#if defined(__APPLE__)
```

Comment:
I wonder if it would make sense to move this block to its own new function?

Reply:
Yeah, it probably would help readability.

Reply:
Done.